braid/braid_defs.h

```diff
@@ -48,6 +48,12 @@ typedef int    braid_Int;
 #define braid_Int_Max INT_MAX;
 #define braid_Int_Min INT_MIN;
 
+/**
+ * Defines byte type
+ * (can be any type, but sizeof(braid_Byte) MUST be 1)
+ */
+typedef char braid_Byte;
+
```

Comment:
From what I can tell, a char is guaranteed to be 1 byte, 
https://www.ibm.com/docs/en/ibm-mq/7.5?topic=platforms-standard-data-types